### ISSUE TITLE
Preview 5.5

### ISSUE THREAD

**Issue by bstone** [djhunter/webwork-open-problem-library#3]
I am not sure what to do with this preview activity. You can find it here:

```
webwork-open-problem-library/Contrib/Westmont/ActiveCalculus/Preview_5_5/preview_5_5_a.pg
```

I have tried to create tables and create choices, but I am not really pleased with it. I think this problem needs to be revised, but I am not sure how. Any suggestions?

**Comment by djhunter** [comment]
Yeah, that's a lot of essay responses to grade. My first impulse is to avoid that. Maybe the way around is to avoid tracking the book's exercise so closely. How about breaking it up into 3 or 4 separate questions:
Question 1: A few of the integrals from parts a-d, with drop down multiple choice questions (u-sub, parts, combo, niether), like your table, but without the followup questions.
Question 2: A few of the integrals from parts a-d that can be done using u-substitution, and students provide the u (graded by webwork, not as an essay).
Question 3: A few of the integrals from parts a-d that can be done using parts, and students provide the u and dv (graded by webwork, not as an essay).
I'm not sure what to do about the "combo" ones, because I think there are multiple reasonable correct answers. Since "combo" was sort of addressed in Question 1, maybe it's OK to just ignore those.
